Result of `nixpkgs-review pr 94240` on darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
<br>- kitty
</details>

```
builder for '/nix/store/cc6c9qb5l1d4frav2cv80cjhv1ir3zrm-kitty-0.18.2.drv' failed with exit code 1; last 10 log lines:
      cache[rtype].remove(name)
  KeyError: '/mp-vkbse8fx'
  Traceback (most recent call last):
    File "/nix/store/rf859gichqk3rvxw7xnlavpc8zalsk62-python3-3.8.3/lib/python3.8/multiprocessing/resource_tracker.py", line 201, in main
      cache[rtype].remove(name)
  KeyError: '/mp-onh27hyr'
  Traceback (most recent call last):
    File "/nix/store/rf859gichqk3rvxw7xnlavpc8zalsk62-python3-3.8.3/lib/python3.8/multiprocessing/resource_tracker.py", line 201, in main
      cache[rtype].remove(name)
  KeyError: '/mp-bkf0meiz'
cannot build derivation '/nix/store/51cgkdi3qv0j1127mwvxwjmxrp2bq82b-env.drv': 1 dependencies couldn't be built
```

--- a/pkgs/applications/misc/kitty/default.nix
+++ b/pkgs/applications/misc/kitty/default.nix
@@ -20,14 +20,14 @@
 with python3Packages;
 buildPythonApplication rec {
   pname = "kitty";
-  version = "0.18.1";
+  version = "0.18.2";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "kovidgoyal";
     repo = "kitty";
     rev = "v${version}";
-    sha256 = "1g4mfgygyl143k0k6d3cb8b2l05ahiamlcqs1iqi66pc73cax4z6";
+    sha256 = "0x6h8g017mbpjkpkb1y8asyfdc48bgjzmj5gachsp5cf5jcqwir2";
   };
 
   buildInputs = [
@@ -85,9 +85,15 @@ buildPythonApplication rec {
 
   checkInputs = [ pillow ];
 
-  checkPhase = ''
-    ${python.interpreter} test.py
-  '';
+  checkPhase =
+    let buildBinPath =
+      if stdenv.isDarwin
+        then "kitty.app/Contents/MacOS"
+        else "linux-package/bin";
+    in
+    ''
+      env PATH="${buildBinPath}:$PATH" ${python.interpreter} test.py
+    '';
 
   installPhase = ''
     runHook preInstall

